src/saga/adaptors/slurm/slurm_job.py

```diff
@@ -387,7 +387,7 @@ def _open(self):
                     '| xargs echo'
         _, out, _ = self.shell.run_sync(ppn_cmd)
         ppn_vals  = [o.strip() for o in out.split() if o.strip()]
-        if len(ppn_vals) == 1: self._ppn = int(ppn_vals[0])
+        if len(ppn_vals) >= 1: self._ppn = int(ppn_vals[0])
```

Comment:
Many systems that have SLURM versions greater that 17.11.5 return multiple values as `ppn`. This is not the best fix, but it selects the first of the returned values.

The issue here is that these systems provide different number of cores per node and setting `_ppn` to the first value will definitely cause, like asking for more nodes as the required